pkg/controller/controller.go

```diff
@@ -322,6 +357,15 @@ func (ctrl *resizeController) resizePVC(pvc *v1.PersistentVolumeClaim, pv *v1.Pe
 		pvc = updatedPVC
 	}
 
+	// if pvc previously failed to expand because it can't be expanded when in-use
+	// we must not try expansion here
+	if ctrl.usedPVCs.hasInUseErrors(pvc) && ctrl.usedPVCs.checkForUse(pvc) {
+		// Record an event to indicate that resizer is not expanding the pvc
+		ctrl.eventRecorder.Event(pvc, v1.EventTypeWarning, util.VolumeResizeFailed,
+			fmt.Sprintf("CSI resizer is not expanding %s because it is in-use", pv.Name))
```

Comment:
Can you add a message to indicate that the CSI driver only supports offline volume expansion.

Reply:
done.